routers/user_router.py

```diff
@@ -34,3 +35,18 @@ def create_user(user: user_schema.UserCreate, db: Session = Depends(get_db)) ->
     db.refresh(new_user)
     return new_user
 
+@router.delete("/users/{user_email}", status_code=status.HTTP_204_NO_CONTENT)
+def delete_user(user_email: str, db: Session = Depends(get_db)):
+    #Check if the user exists
+    user = db.query(User).filter(User.email == user_email).first()
+    
+    if user is None:
+        # if User does not exist, raise a 404 error
+        raise HTTPException(status_code=404, detail = "User does not exist")
+    
+    #Delete existing user from db
+    db.delete(user)
+    db.commit()
+    
+    return None
```

Comment:
instead of returning None, it would be great to return a message like  "User with email $email was successfully deleted"